client/src/main/java/tds/session/error/ValidationErrorCode.java

```diff
@@ -0,0 +1,8 @@
+package tds.session.error;
+
+
+public class ValidationErrorCode {
+    public static final String PAUSE_SESSION_IS_CLOSED = "sessionClosed";
+    public static final String PAUSE_SESSION_OWNED_BY_DIFFERENT_PROCTOR = "ownedByDifferentProctor";
+    public static final String PAUSE_SESSION_ACCESS_VIOLATION = "accessViolation";
```

Comment:
These are the codes that will be translated to messages at some point on the client side, correct?  I would like the codes to be easy to understand what they are referring to.  I wonder if "namespacing" them makes sense, or prefixing them with the action or area they refer to.  I think it really depends on how many of these we think there will be and if it will get unweildy to look at them in the database and know what they mean.

In the past I've done things like `"session:ownedByDifferentProctor"` and `"session:accessViolation"`.  And if accessViolation is more general than this section, maybe it becomes `"general:accessViolation"`

I'm guessing this is a bigger conversation than this PR, so it should not hold anything up here.